genai-perf/genai_perf/config/input/config_analyze.py

```diff
@@ -34,10 +34,10 @@ def __init__(self) -> None:
         # yapf: disable
         sweep_parameter_template_comment = \
         (f"Uncomment the lines below to enable the analyze subcommand\n"
-         f"For further details see analyze.md sweep_parameters:\n"
-         f"  concurrency:\n"
-         f"    start: {AnalyzeDefaults.MIN_CONCURRENCY}\n"
-         f"    stop: {AnalyzeDefaults.MAX_CONCURRENCY}")
+         f"#  For further details see analyze.md sweep_parameters:\n"
```

Comment:
This is just a formatting change to ensure that if the user also uncomments this line the template still works.